src/config/rules-axe-options.js

```diff
@@ -0,0 +1,17 @@
+const rulesAxeOptions = {
+  /**
+   * Rule: role attribute has valid value
+   * Url: https://act-rules.github.io/rules/674b10
+   *  - Disable below checks
+   *    - `fallbackrole`
+   *    - `invalidrole`
+   */
+  '674b10': {
```

Comment:
Why are you configuring this per rule, instead of just having a global config that you use for every rule? Seems unnecessarily complex.